### PR TITLE
Accessibility: Add link to alternative format version of exhibit

### PR DESCRIPTION
**Pull request by doana**
Hi all, 

This PR adds an additional field to the exhibit table and a corresponding field on the edit form. This field allows the creator so specify a URL where an alternative format version of the exhibit can be found. When a link is provided, a hidden link is rendered at the top of the public exhibit page, directing assistive technology users to the supplied URL. This hidden link is not visible to sighted users, but is available for users who are relying on assistive technology to view the page.

I have done this work because legislation in my country requires all of our web content to meet the [WCAG 2.0](https://www.w3.org/TR/WCAG20/) standard for accessibility. Specifically this work relates to [Guideline 1.1 of WCAG 2.0](https://www.w3.org/WAI/WCAG20/quickref/#text-equiv).

Hopefully this change should be fairly unobtrusive for most users. The link is not rendered on the public exhibit page when there is no value provided for the accessible URL field.

I had to bump the version number to 2.5.2 so I could have a new version to target for the DB migrations. If that version number isn't consistent with your versioning practices it may need to be changed.

Hopefully I've covered all of my bases here and things should are up to your standards. Please let me know if I can do anything else to help!

-Adam

### CODE REVIEW

**Comment by jeremyboggs** [comment]
Hi @doana, thanks for this pull request! Very sorry we've left it sitting without a response for so long. I think this is a terrific idea, and appreciate your work on it. 

Where might a user expect to put this accessible content? Right now, from a cursory glance over your commits, it could be at any URL, and not necessarily within the Omeka site itself.

Could you share links to examples where you've implemented something similar? I'd love to see the differences in presentation between the default Neatline exhibit and your accessible version. If it's possible, and conducive, I could really see automatically building an accessible rendition directly from Exhibit text. Or at the very least, providing a composition space within the Neatline editor to write this content.

**Comment by doana** [comment]
Hey! Thanks for following up!

My plan for now was just to have them create a simple page (Simple Pages plugin) with the alternative content and use that.

I have this code in production on a couple of sites, though no one has published any completed Neatline exhibits at this point. 

In terms of the visual presentation this should be identical to the default version. In the Neatline show template this PR adds a link just above the exhibit, but that link is hidden (positioned off screen) with CSS. That way screen readers will still see the link, but it should look the same to a visual user. 

Generating the content would be a great value-add, but I think it would be important that the user would be able to edit the generated result. That way users could ensure that the generated markup is itself accessible and that the generated content is actually the "equivalent" of the exhibit itself. 

A composition space would be nice and significantly easier to implement.

**Review by jeremyboggs — APPROVED** [review]

**Comment by jamiefolsom** [comment]
Hey @doana! 

We're working toward a patch release this week -- we've tested your PR, and it's looking good.

If you have a few minutes, would you: 

1. Rebase your branch against the current master branch on http://github.com/scholarslab/neatline, 
2. Squash commits down to just one, and
3. Update your PR with that rebased version?

If so, we can include this PR in the release.

Let me know, and thanks!

File: RELEASE-LIST.md [hunk 1]
```diff
@@ -35,6 +35,8 @@ Once all of that is in place, we're ready to go.
 
 - Hallway test the zip.
 
+  - Is the CKEditor in `views/shared/javascripts/dist/production/ckeditor`?
```
Comment (jamiefolsom) [inline RELEASE-LIST.md:38]: @doana can you elaborate a bit on this addition? Have you run into CKEditor being missing?

**Review by jamiefolsom — COMMENTED** [review]
Looks good. Just one question.

File: RELEASE-LIST.md (hunk 1 above)
Comment (jamiefolsom) [reply RELEASE-LIST.md:38]: Never mind, @doana, it's been pointed out to me that that commit is not yours.

**Pull request merged**